src/cdc/runner.zig

```diff
@@ -1424,7 +1424,7 @@ test "amqp: ProgressTrackerMessage" {
 }
 
 test "amqp: JSON message" {
-    const Snap = @import("../testing/snaptest.zig").Snap;
+    const Snap = stdx.testing.snaptest.Snap;
```

Comment:
One of the guidelines for `stdx` is that it should provide a flat API, so lets make this just `stdx.Snap`